getstarted.rst

```diff
@@ -1,77 +1,119 @@
+
+Each nio project will be in its own directory too. We will start by creating some directories for these.
+
+.. code-block:: bash
+
+    cd
+    mkdir nio
+    cd nio
+    mkdir projects
+    mkdir versions
 
 Installation
 ------------
 
-We will be installing nio to a virtual environment. This keeps nio and all its dependencies isolated from other python projects and environements you may already have on your machine. You can read more about them at `here <http://docs.python-guide.org/en/latest/dev/virtualenvs/>`_.
+Lets start by installing nio version 1.5.1.
 
 .. code-block:: bash
 
-    $ mkdir nio
-    $ cd nio
-    $ virtualenv env
-    $ source env/bin/activate
+    cd ~/versions
+    virtualenv-3.4 1.5.1
+
+If virtualenv was installed with a python version less than 3.4, then you will need to specify the python version when creating your virtual environtment. To check which version of python virtualenv will use, type `virtualenv -h` and look at what version is specifed in the `-p` section of the docs.
+
+.. code-block:: bash
 
-When you're done using nio, you can leave the virtual environment with `deactivate`. Be sure to always come back to this directory and run `source env/bin/activate` when you are going to be running nio.
+    virtualenv-3.4 -p python3 1.5.1
 
-Download the latest nio and nioext wheels from the `nio Versions Page <http://n.io/versions>`_. Then install them using pip.
+You now need to activate your virtual environment. This will add a `(1.5.1)` to the beginning of your command line to indicate the virtual environemnt that you are using.
 
 .. code-block:: bash
 
-    $ wget http://n.io/versions/nio/nio-1.5.0-py3-none-any.whl
-    $ wget http://n.io/versions/nioext/nioext-1.5.0-py3-none-any.whl
-    $ pip install http://n.io/versions/nio/nio-1.5.0-py3-none-any.whl
-    $ pip install http://n.io/versions/nioext/nioext-1.5.0-py3-none-any.whl
+    source 1.5.1/bin/activate
+
+When you're done using nio, you can leave the virtual environment with `deactivate`. When using nio again, be sure to activate the virtual environemtn first with `source ~/nio/versions/1.5.1/bin/activate`.
+
+OK, now we can finally install `nio <http://n.io/versions/nio/>`.
+
+.. code-block:: bash
+
+    pip install http://n.io/versions/nio/nio-1.5.1-py3-none-any.whl
+    pip install http://n.io/versions/nioext/nioext-1.5.1-py3-none-any.whl
 
 The installation of nio is now complete! You can run the instance from a project directory with the ``run_nio`` command. See :ref:`setting-up-a-project` for instructions on creating a project directory.
 
+We now install a tool that helps with common operations like creating projects, adding blocks and managing dependencies. It is called the nio Command Line Interface (CLI).
+
+.. code-block:: bash
+
+    pip install https://github.com/neutralio/nio-cli/archive/master.zip
```

Comment:
I would install this before activating the virtualenv -- otherwise it is going to be installed into your virtualenv!